Package.swift

```diff
@@ -1,10 +1,22 @@
-// swift-tools-version:3.1
+// swift-tools-version:4.0
 import PackageDescription
 
 let package = Package(
   name: "Stencil",
+  products: [
+        .library(name: "Stencil", targets: ["Stencil"]),
+    ],
```

Comment:
OCD-me speaking: seems like there's inconsistent indentation here compared to the rest of the file (tab vs space probably?)

Reply:
Fixed 👍